### PR TITLE
P2022 1320 register screen/create layout

### PR DESCRIPTION
**Pull request by Anna-Surma**
Add RelativeLayout with 3 EditText and a Button.
Navigate from SingUpFragment to MainActivity by click singUp button.

image

### CODE REVIEW

**Comment by Lucas-Fresko** [comment]
This holds a copy of the commit from your previous PR. I had that problem before, would be good to figure out the cause and solution. I resorted to cloning a clean repository. Did you push this via Android Studio git gui?

**Comment by Anna-Surma** [comment]
> Did you push this via Android Studio git gui?

Yes, I'll try to change that

File: app/src/main/java/com/intive/patronage22/intivi/Adapter/FragmentsAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+import com.intive.patronage22.intivi.Fragments.SignInFragments
+import com.intive.patronage22.intivi.Fragments.SignUpFragments
+
+internal class FragmentsAdapter (fm: FragmentManager, var totalTabs: Int): FragmentPagerAdapter(fm) {
+
+    override fun getItem(position: Int): Fragment {
+        return when(position){
+            0-> {
```
Comment (kosiara) [inline app/src/main/java/com/intive/patronage22/intivi/Adapter/FragmentsAdapter.kt:13]: we could create an enum class to encapsulate those constant values...
e.g. 

enum class ActiveTab {
    SIGN_IN, SIGN_UP
}

File: app/src/main/java/com/intive/patronage22/intivi/Fragments/SignInFragments.kt [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.EditText
+import com.intive.patronage22.intivi.MainActivity
+import com.intive.patronage22.intivi.R
+
+import com.intive.patronage22.intivi.databinding.FragmentSignInBinding
+
+class SignInFragments : Fragment() {
```
Comment (kosiara) [inline app/src/main/java/com/intive/patronage22/intivi/Fragments/SignInFragments.kt:15]: We usually use singular for fragments... e.g. `SignInFragment`

**Pull request merged**